src/features/dashboard/components/Dashboard.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from "react";
+import ProjectCard from "./ProjectCard";
+import StatusFilter from "./StatusFilter";
+import { getProjects, getDocumentsByProject } from "../api/api";
+
+interface Document {
+  name: string;
+  project: string;
+  schema: string;
+  document_edit_state: string; 
+}
+
+interface Project {
+  id: number;
+  title: string;
+  schema: string;
+  team: string;
+  progress: number;
+  documents: {
+    ongoing: Document[];
+    open: Document[];
+    completed: Document[];
+  };
+}
```

Comment:
```suggestion
interface Document {
  name: string;
  project: Project;
  schema: Schema; // you might need to add a new Schema Interface for this
  document_edit_state: string; 
}

interface Project {
  id: number;
  title: string;
  schema: string;
  team: Team; // you might need to add a new Team Interface for this
  progress: number;
  documents: {
    ongoing: Document[];
    open: Document[];
    completed: Document[];
  };
}
```

This should be outsourced to a `model` folder, as these interfaces can be reused in different Views

The Backend is currently refactoring the api to provide nested object, e.g. the `Team` and `Project` i noted above